pkgs/tools/archivers/ratarmount/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, indexed-zstd
+, indexed_gzip
+, indexed_bzip2
+, fusepy
+, lzmaffi
+}:
+
+buildPythonPackage rec {
```

According to the readme, this is meant to also be used as an application.

More information on this can be found [here](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#topythonapplication-function)

```suggestion
buildPythonApplication rec {
```